input/posts/en/transfer_learning.md

```diff
@@ -0,0 +1,37 @@
+﻿title: Transfer learning
```

Comment:
change title and file name to something more specific (information about shapes and alexnet should be included)